manifests/r/RedHat/VirtIO/0.1.262-2/RedHat.VirtIO.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: RedHat.VirtIO
+PackageVersion: 0.1.262-2
+PackageLocale: en-US
+Publisher: Red Hat, Inc.
+PackageName: Virtio-win-guest-tools
```

```suggestion
PackageName: Virtio-win-guest-tools
PackageUrl: https://github.com/virtio-win/virtio-win-pkg-scripts/blob/master/README.md#downloads
```

`PackageUrl` is required for the manifest.